CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ Please see [CONTRIBUTING.md](https://github.com/cucumber/cucumber/blob/master/CO
 
 ### Removed
 
+* Support for running Cucumber in web browsers has been removed ([#1508](https://github.com/cucumber/cucumber-js/pull/1508))
```

Comment:
Wondering if we need to qualify this so people don't get confused and think they can't test with e.g. webdriver any more?

Reply:
Yes, this can definitely be misunderstood!

Reply:
And should we explain **why** we drop support for browser runs?

Reply:
Had a go at a more informative message, let me know what you think.

Reply:
I like it, thanks!